code/1_cleaning/07_merge_grace_gaezpy.R

```diff
@@ -0,0 +1,97 @@
+# Description: This file calculates the crop-specific and across-crop productivity
+# of each GRACE cell using potential yields data from GAEZ.
+
+# ENVIRONMENT SETUP ------------------------------------------------------------
+
+# run setup script
+source("code/0_env_setup/00_setup.R")
+
+# CLEAN GRACE DATA -------------------------------------------------------------
+
+# load GRACE data
+grace <- read_dta("data/input/grace/grace.dta")
```

Comment:
out of curiosity - what is this file? is it created in our main paper pipeline?

Reply:
It is basically analysis-ready + the datasets you gave me (cereal production and population data). I created it in the main paper pipeline under `3_water_crops_facts/aeapp/code/06_repl_data` in the `pp_replication` branch.